frontend/src/store/index.js

```diff
@@ -165,10 +181,13 @@ export function createStore() {
           });
         },
         /* User */
-        getUser: (store, userId) => {
-          api.getUser(userId).then((user) => {
-            return store.commit('setUser', user);
-          });
+        getUser: (store) => {
+          const user = {
+            id: localStorage.id_token,
+            name: localStorage.user_name,
+            picture: localStorage.profile_thumbnail,
```

Comment:
Ideally we wouldn't be accessing `localStorage` directly and would export some User object in auth... but that can be done in the future.

Reply:
Yep. I can add a `TODO` here.